install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -525,6 +525,11 @@ $ oc patch storageclass standard -p '{"metadata": {"annotations": \
     {"storageclass.kubernetes.io/is-default-class": "true"}}}'
 ----
 
+[NOTE]
+====
+Only one StorageClass can be set as default. If more than one default is marked, a PVC cannot be created without explicitly specifying the `storageClassName`.
```

Comment:
1. s/Only one StorageClass can be set as default/Only one StorageClass should be set as default/
2. A PVC still can be created when there are more than on default storage class. It is that no PV and backend volume dynamic provisioned to bind with it.
So we need to re-organized the sentences.

Reply:
Trying to capture comment and K8s docs to make this more clear. Not sure if I understood this correctly:  "...no PV and backend volume dynamic provisioned to bind with it."

Revised: "If more than one StorageClass is marked as default, a PV cannot be dynamically provisioned. With more than one default, a PVC can only be created if the `storageClassName` is explicitly specified. Therefore, only one StorageClass should be set as the default."

Another option is to shorten to: "If more than one StorageClass is marked as default, a PVC can only be created if the `storageClassName` is explicitly specified. Therefore, only one StorageClass should be set as the default."

Reply:
@bobfuru The shorten one looks great.